### ISSUE TITLE
getActivitySamples activityName is unknown

### ISSUE THREAD

**Issue by j-carvalho** [StasDoskalenko/react-native-google-fit#224]
I am not sure if it's related to the library or the actual data in GoogleFit.

But I've tried to add some activities into GoogleFit, using the built-in activities.

But when execution this function, even thought the data is indeed received, the activityName is always unknown, which makes it hard to identify in the app for which activity this entry actually is.

Not sure if its related with the particular implementation of getActivitySamples. I am interesting in getting different types of activities, e.g. stair climbing / meditation.

e.g.

```json
[{"activityName": "unknown", "calories": 12826.4697265625, "device": "", "end": 1620049320000, "sourceId": "com.google.android.gms", "sourceName": "", "start": 1619388000000, "tracked": false}, {"activityName": "unknown", "calories": 36961.6171875, "device": "", "end": 1622018220000, "sourceId": "com.google.android.gms", "sourceName": "", "start": 1620120480000, "tracked": false}, {"activityName": "unknown", "calories": 51137.13671875, "device": "", "end": 1624658399999, "sourceId": "com.google.android.gms", "sourceName": "", "start": 1622021820000, "tracked": true}]
```

**Comment by aboveyunhai** [comment]
Check the doc , https://developers.google.com/fit/rest/v1/reference/activity-types.
There is a `unknown` when Google failed to figure the activity. I believe it's just a google side problem since we are just getting the data from it. 
Just keep watching if the same problem appears again or try to find the same activity inside official GoogleFit app to see if it exists with a valid name. If so, then it might be something wrong with our side.

**Comment by storm2513** [comment]
Same issue. I have set up activity type and activity name in google fit app, but receive "unknown" as activity name.

**Comment by aboveyunhai** [comment]
> Same issue. I have set up activity type and activity name in google fit app, but receive "unknown" as activity name.

In order to help out the issue, please state which unknown activity you had setup, the correct name, Even better, does it appear correctly in official Googlefit app.
Repeat same issue wouldn't help out the problem at all.

**Comment by j-carvalho** [comment]
@aboveyunhai the issue I had was with the activity called Meditation, for example (activity type 45).

I added a manual entry in the GoogleFit app for this activity. 

And then I tried to query the activity samples and it is included but activityName unknown.

**Comment by aboveyunhai** [comment]
@j-carvalho  I was able to get the meditation after adding it by the GoogleFit app.
I believe the `unknown` type might come from a combination of multiple activity with in one time span, so Google Fit Api does not able to distinguish it hence give you a `unknown` type.
![image](https://user-images.githubusercontent.com/35160613/120564391-cdeac700-c3d8-11eb-8400-86bf3010835e.png)
If we noticed the sourceId, it's from `com.google.android.gms`, we probably need to figure out what it is.
